lib/share.js

```diff
@@ -68,8 +68,11 @@ module.exports = function sync (type, opts, dat) {
       // File Imports
       progressOutput[2] = 'Importing new & updated files to archive...'
 
-      // TODO: allow live: true
-      importStatus = dat.importFiles({live: false, resume: true}, function (err) {
+      importStatus = dat.importFiles({
+        live: false, // TODO: allow live: true. opts.live is used by archive.live though =(
```

Comment:
> So watching files still doesn't work well here with sync?

There is nothing to stop it from working if we [just added it right now](https://github.com/joehand/dat-next/commit/abec1eba0dd9b1c7202c60e655326d8defbc60e4#diff-3ceaa0080b7852c167b868be66fb9206R71). But because of the bug issues we saw last time, I want to think through the UI a lot more.

I've been thinking about it but should outline what needs to be done in the open issue. Basically:
* hyperdrive-file-importer needs events for live file updates (to differentiate from non-live)
* need to make sure hyperdrive-stats is compatible (think it is but recall some issue)
* rethink importing UI for live updates (instead of the bar)

Before we were trying to increment the importer progress as updates happen but that was too messy. Instead I think it'd be good to show the last 5 files imported or something and list them as more updates happen.